### DISCUSSION TITLE
Compare two dates with Validator

### DISCUSSION THREAD

**Discussion by gribbl** [adonisjs/core#3643] [Help] [answered]
Hi,

I was wondering if it was possible to compare 2 dates with Validator. 

Let's say I have a start and end date. The goal is to check that the start date is not greater than the end date and vice versa.

**Comment by Julien-R44** [comment, marked as answer]
Hello 👋 

Yes it is possible, take a look here :
https://docs.adonisjs.com/reference/validator/rules/after
https://docs.adonisjs.com/reference/validator/rules/before

Especially : 
https://docs.adonisjs.com/reference/validator/rules/after-field
https://docs.adonisjs.com/reference/validator/rules/before-field
